background.js

```diff
@@ -10,6 +10,7 @@ $(document).ready(function () {
   var talking = true;
   var recognition;
   var txt;
+  var id = 1; //for screenshots
```

Comment:
I'm not so sure about this

Reply:
It's there to handle the usecase when user requests multiple screenshots in single session, so as to uniquely identify each of them.